### ISSUE TITLE
resnet34 compression result

### ISSUE THREAD

**Issue by jsleeg98** [jsleeg98/ReconstructionPruning#22]
![Acc_test (6)](https://user-images.githubusercontent.com/64693974/193415247-1565a7a1-69df-413c-8789-bf5382ef6308.svg)

optimizer = optim.SGD(model.parameters(), lr = 0.01, momentum=0.9, weight_decay=0.01)
scheduler = optim.lr_scheduler.MultiStepLR(optimizer, milestones=[60, 80], gamma=0.1)

|compression ratio|method|best acc|
|--|--|--|
|original|-|93.34%|
|0.3|l1 magnitude|90.02%|
|0.3|l2 magnitude|89.85%|
|0.5|l1 magnitude|89.05%|
|0.5|l2 magnitude|89.42%|
|0.7|l1 magnitude|88.03%|
|0.7|l2 magnitude|87.27%|
|0.9|l1 magnitude|78.93%|
|0.9|l2 magnitude|78.89%|

**Comment by jsleeg98** [comment]
README에 모두 정리
